Common/AR/qml/CalibrationSlider.qml

```diff
@@ -0,0 +1,70 @@
+/*******************************************************************************
+ *  Copyright 2012-2019 Esri
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ******************************************************************************/
+
+import QtQuick 2.12
+import QtQuick.Controls 2.2
+
+Item {
+    id: root
+    implicitWidth: label.implicitWidth + slider.implicitWidth
+    implicitHeight: Math.max(label.implicitHeight, slider.implicitHeight)
+
+    // The text to display on the left of the slider.
+    property alias text: label.text
+
+    // The width of the text.
+    property alias textWidth: label.width
+
+    // The offset to applied.
+    property double value: 0.0
```

Comment:
some of these properties can be marked readonly - https://doc.qt.io/qt-5/qtqml-syntax-objectattributes.html#read-only-properties

Reply:
No, all the property can be changed. `value` can be set to 0.0 when `Reset calibration` is called.